packages/biome-config/README.md

```diff
@@ -0,0 +1,41 @@
+# @readme/biome-config
+
+Core coding standards for ReadMe projects using [Biome](https://biomejs.dev/).
+
+[![](https://raw.githubusercontent.com/readmeio/.github/main/oss-header.png)](https://readme.io)
+
+[![npm](https://img.shields.io/npm/v/@readme/biome-config)](https://npm.im/@readme/biome-config) [![Build](https://github.com/readmeio/standards/workflows/CI/badge.svg)](https://github.com/readmeio/standards)
+
+## Installation
+
+You'll need to install [Biome](https://biomejs.dev/) into your project. Use this shortcut to install them alongside the config:
+
+```sh
+npx install --dev @biomejs/biome
```

Comment:
is this a thing or did you mean this (see below)?

```suggestion
npx install-peerdeps --dev @readme/biome-config
```

Reply:
I'm pretty sure `npm install` installs peer deps by default now. That was something that changed in NPM 7.

https://docs.npmjs.com/cli/v9/commands/npm-install#strict-peer-deps
https://stackoverflow.com/a/35207983

Reply:
oh. 🙃